repoSets.json

```diff
@@ -95,6 +95,11 @@
         "repo": "DataProtection",
         "inclusionLevel": "AllItems"
       },
+      {
+        "org": "aspnet",
+        "repo": "DependencyInjection",
```

Comment:
The repo is already listed in the `kcore` section, but it's listed as  ItemsAssignedToPersonSet (or whatever). So, all that needs to happen is to change it from ItemsAssigned to AllItems.

Reply:
I see, thanks !